src/build.js

```diff
@@ -69,19 +71,34 @@ export const build = function(statics) {
       } else if (char === '"' || char === "'") {
         quote = char;
         buffer += char;
-      } else if (charHead === '}}' && str[j + 2] !== '}') {
+      } else if (expr === EXPR_SPECIAL_COMMENT && charHead === '--' && str.substr(j + 2, 2) === '}}') {
+        commit();
```

Comment:
looks good! try changing this one to `commit('');`. 
the commit function ignores empty buffer.

Reply:
That didn't help. Any other ideas?

Reply:
this line might have caused the issue
https://github.com/luwes/hyperstache/pull/6/files#diff-01994488c699366e7146092d8327cadaR194

added a fix for it. thanks again for the contribution!
we should be able to merge it soon.